.travis.yml

```diff
@@ -59,10 +59,14 @@ script:
 jobs:
   fast_finish: true
   allow_failures:
```

Comment:
I didn't quite understand what this block does so I just followed the pattern here.